src/hooks/useParam.ts

```diff
@@ -0,0 +1,61 @@
+import { useRouter } from 'next/router';
+
+export enum RouteLocation {
+    Sykmeldt,
+    Sykmelding,
+    Soknad,
+}
+
+interface SykmeldtRoute {
+    sykmeldtId: string;
+    location: RouteLocation.Sykmeldt;
+}
+
+interface SykmeldingRoute {
+    sykmeldtId: string;
+    sykmeldingId: string;
+    location: RouteLocation.Sykmelding;
+}
+
+interface SoknadRoute {
+    sykmeldtId: string;
+    soknadId: string;
+    location: RouteLocation.Soknad;
+}
+
```

Comment:
Denne abstraheringen er mest for å unngå utvikler feil. Når man bruker next sitt API direkte:

```
const { sykmeldtId } = useRouter();
```

så er typen til `sykmeldtId` `string | string[] | undefined`, og dersom man skriver `sykmeldtid` så sitter man å klør seg i hodet. Så det er mest for å unngå teite feil, fordi pathsa er jo basert på filstrukturen, og path-paremeterene er basert på filnavnene som `[sykmeldtId].tsx`